docker-entrypoint.sh

```diff
@@ -60,7 +60,13 @@ wait_for_db() {
 	done
 }
 
+escape_for_replace() {
+	printf '%s\n' "${1}" | sed -e 's/[\/&]/\\&/g'
```

Comment:
Is this still needed? (Sorry I am on mobile, reading complete code-files is hard)

Reply:
I kept it in case we need it for future developments, but it's currently not used so it can be removed.